tests/test_accel.py

```diff
@@ -1066,13 +1066,11 @@ def test_reverse():
 
 
 @timer
-def test_maneuvers():
-    # three maneuvers, Hohman transfer, inclination change, bielliptic transfer
-    # hohman transfer from LEO to GEO; test that eccentricity and
-    # final semimajor axis are close to expectations.
-    # inclination change: do tiny burn to change inclination.  Make
-    # sure that di ~ dv * n * a, and that a and e don't change much.
-    # first a Hohman transfer
+def test_Hohmann_transfer():
+    """
+    Test Hohmann transfer between Low Earth Orbit (LEO) and Geostationary Orbit (GEO).
+    Check that (a, e, i) are close to expectations.
```

Comment:
Please say what the variables `a`, `e`, and `i` represent. We want to be as pedantic as possible so that future developers don't need to go searching for domain specific information.